packages/ui/src/views/components/sidebar/Sidebar.tsx

```diff
@@ -97,6 +97,7 @@ export function Sidebar() {
             visible: false,
         };
 
+        sidebarService.options.visible = false;
```

Comment:
This is awkard. `SidebarService` should not expose its property for other modules to change it directly. Should refactor later.